lib/fluent/plugin/out_influxdb.rb

```diff
@@ -132,6 +132,12 @@ def write(chunk)
         tags[@sequence_tag] = @seq
         @prev_timestamp = timestamp
       end
+
+      if values.empty?
+          $log.warn "Skip record '#{record}', because InfluxDB requires at least one value in raw"
```

Comment:
`log` is correct.
Don't use `$log` inside `write` for per plugin level logging.